curriculum/challenges/english/25-front-end-development/workshop-medical-data-validator/684fd85424ffdb2edff3afd1.md

```diff
@@ -0,0 +1,127 @@
+---
+id: 684fd85424ffdb2edff3afd1
+title: Step 26
+challengeType: 20
+dashedName: step-26
+---
+
+# --description--
+
+Now you can see `{'patient_id': None}` printed to the terminal because the lowercase `p` does not match `P1001` and the `and` operator returns the first falsy value of the expression.
+
+You want to ensure that the patient id starts with the letter `p`, but it can be either lowercase or uppercase. To modify the matching behavior of regular expressions, you can use flags. For example, `re.search` accepts a third argument to specify any flags:
```

Comment:
```suggestion
You want to ensure that the `patient_id` starts with the letter `p`, but it can be either lowercase or uppercase. To modify the matching behavior of regular expressions, you can use flags. For example, `re.search` accepts a third argument to specify any flags:
```

or 

```suggestion
You want to ensure that the patient ID starts with the letter `p`, but it can be either lowercase or uppercase. To modify the matching behavior of regular expressions, you can use flags. For example, `re.search` accepts a third argument to specify any flags:
```